### ISSUE TITLE
Screen Reader Compatibility - Submitted by Jinho Hwang

### ISSUE THREAD

**Issue by connordear** [connordear/eecs2311group2#1]
# Screen reader cannot read anything in the program.

## Summary
NVDA failed to read any components in the program except the window.

## Technical Information
**OS** : Windows 10 Home  
**Java Version** : version 8 update 161 Build 1.8.0_161-b12  
**Used Screen Reader** : NVDA Version 2017.4  
**Used Release Version** : vM  

## Description
Screen reader NVDA has failed to read any of the components within the window when it is focused and traversed with "tab" button. The default 

# Steps to Reproduce

[Main Menu](#screenshots) in **Figure 1**
1. Open the Braille Author.
2. Open NVDA screen reader.
3. Traverse buttons by pressing Tab.
4. Press "New" button by pressing enter when it is focused.

["Scenario Editor title, cell, number inputter"](#screenshots) in **Figure 2**
1. On the main menu, press "New" button.
2. Traverse buttons and text fields by pressing Tab.

[Scenario Editor](#screenshots) in **Figure 3**
1. Traverse Add by pressing Tab.
2. Click "Add" button.
3. Traverse "Delete", "Up", "Down", "Save", and "Run" buttons.
4. Click "Up" button.
5. Click "Down" button.
6. Click "Delete" button.
7. Click "Save" button.
8. Click "Run" button.
9. Click "Add" button twice.
10. Tab through.

## Expected Results
Expected for NVDA to read the focused elements. 

## Actual Results
None of the interaction, except for the file chooser, were readable by NVDA.

## Screenshots

![Image of the main menu](https://i.imgur.com/HTANOSR.png)
**Figure 1**. The main menu. 1 - 4 are unable to read by NVDA.

![Image of Scenario maker](https://i.imgur.com/70Iyin7.png)
**Figure 2**. "Scenario Editor title, cell, number inputter". 1-4 are unable to read by NVDA.

![Image of Scenario Editor](https://i.imgur.com/w7Yp1wr.png)
**Figure 3**. Scenario Editor. The 1 is unreachable by traversal with tab button, 2 and 3 are unreadable by NVDA.

**Comment by connordear** [comment]
Working as of June 25 2018
